src/index.js

```diff
@@ -34,6 +34,10 @@ module.exports = {
       children: 'ignore',
       props: 'always'
     }],
+    'react/jsx-curly-newline': ['error', {
+      multiline: 'forbid',
+      singleline: 'forbid'
+    }],
```

Comment:
You can pass `'never'` as a string option instead of the explicit object for the same effect